.gitignore

```diff
@@ -47,6 +47,7 @@ nebula/libnebula.a
 src/mozillavpn_autogen/
 /build-*/
 /build/
+dummybuild
```

Comment:
where does this dummybuild come from?

Reply:
cmake command ran before being able to run a test

cmake -S . -B $(pwd)/dummybuild -DBUILD_DUMMY=ON 
cmake --build $(pwd)/dummybuild -j$(nproc)

Reply:
functional test*

Reply:
Right but you can call this dir `build` too. Why dummybuild?

Reply:
Is it written in the README.md?